specs/user_spec.rb

```diff
@@ -38,25 +39,74 @@
   describe "trips property" do
     before do
       @user = RideShare::User.new(id: 9, name: "Merl Glover III",
-                                  phone: "1-602-620-2330 x3723", trips: [])
-      trip = RideShare::Trip.new(id: 8, driver: nil, passenger: @user,
-                                 start_time: Time.parse("2016-08-08"),
-                                 end_time: Time.parse("2016-08-09"),
-                                 rating: 5)
+        phone: "1-602-620-2330 x3723", trips: [])
+        trip = RideShare::Trip.new(id: 8, driver: nil, passenger: @user,
+          start_time: Time.parse("2016-08-08"),
+          end_time: Time.parse("2016-08-09"),
+          rating: 5)
 
-      @user.add_trip(trip)
-    end
+          @user.add_trip(trip)
+        end
 
-    it "each item in array is a Trip instance" do
-      @user.trips.each do |trip|
-        expect(trip).must_be_kind_of RideShare::Trip
-      end
-    end
+        it "each item in array is a Trip instance" do
+          @user.trips.each do |trip|
+            expect(trip).must_be_kind_of RideShare::Trip
+          end
+        end
 
-    it "all Trips must have the same passenger's user id" do
-      @user.trips.each do |trip|
-        expect(trip.passenger.id).must_equal 9
+        it "all Trips must have the same passenger's user id" do
+          @user.trips.each do |trip|
+            expect(trip.passenger.id).must_equal 9
+          end
+        end
       end
-    end
-  end
-end
+
+      describe 'net expenditures' do
+        before do
+          @user = RideShare::User.new(id: 9, name: "Tad Melverno",
```

Comment:
For all the new `User` methods you're missing an interesting test case: what happens if this user has no trips?